### DISCUSSION TITLE
MicroSD Extension Panel - USB C Port inset to far to be usefull

### DISCUSSION THREAD

**Discussion by DannyKlenz** [jon-harper/OmniBox#101] [General]
I purchased the recommended USB-C extension for this panel but when installed the port is far to deep into the panel for any USB-C cable to plug into it. The screws are also interfering with with cable insertion. Its quite possible Im doing something wrong. Any Thoughts?

![IMG20230316141041](https://user-images.githubusercontent.com/124940261/225715962-53082a7d-d8bb-4a0d-b092-e1957aac53a2.jpg)
![IMG20230316141028](https://user-images.githubusercontent.com/124940261/225716289-96bd189c-1db2-4541-9929-00285e8cc632.jpg)

**Comment by jon-harper** [comment]
Do you have calipers handy to measure the depth? That should be 1.6mm deep, but pictures may deceive my eyes so I can't tell how deep that is.

If it is the right depth but you're still having issues, I'll shave that down to 1mm and see if it helps.
